modules/passkey/contracts/test/MockContractImport.sol

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+/* solhint-disable no-global-import */
+pragma solidity >=0.8.0;
+
+import "@safe-global/mock-contract/contracts/MockContract.sol";
```

Comment:
Since we import the `MockContract`, we sould probably rewrite the P-256 verifier tests to use it instead of the `BadP265Verifier` contract.

Reply:
(not in this PR - just a side note).